### PR TITLE
no result to parse on success

### LINKED ISSUE #27245: Account console does not correctly treat link / unlink account
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/ui

### Describe the bug

Upon unlink of the last account, the message is not tranlated.

image

Upon unlink a account when there are multiple accounts linked, action is performed but error is reported.

image

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Errors are reported correctly and only when relevant.

### Actual behavior

Errors are reported incorrectly.

### How to Reproduce?

Go to account console and try unlinking linked accounts.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #27245
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

File: js/apps/account-ui/src/account-security/AccountRow.tsx [hunk 1]
```diff
@@ -31,6 +35,7 @@ export const AccountRow = ({ account, isLinked = false }: AccountRowProps) => {
     try {
       await unLinkAccount(context, account);
       addAlert(t("unLinkSuccess"));
```
Comment (hmlnarik) [inline js/apps/account-ui/src/account-security/AccountRow.tsx:37]: Should this be
```suggestion
      addAlert(t("unLinkSuccess"), AlertVariant.success);
```
?

Comment (jonkoops) [reply js/apps/account-ui/src/account-security/AccountRow.tsx:37]: No need, this is already the default value.

**Review by jonkoops — COMMENTED** [review]
I feel like this should be fixed on the API level. The API here is not setting the content type to JSON, but it is returning a JSON parsable string.

**Comment by edewit** [comment]
@jonkoops actually the response is empty on a delete I think that is fine

**Review by jonkoops — APPROVED** [review]

**Review by hmlnarik — APPROVED** [review]

**Comment by jonkoops** [comment]
Can we do a backport of this for 24?

**Pull request merged**